### ISSUE TITLE
"batch_norm_param" py-faster r-cnn error

### ISSUE THREAD

**Issue by peternara** [XiaozhiChen/resnet-generator#2]
hi.
i have a problem. 
training 
[libprotobuf ERROR google/protobuf/text_format.cc:274] Error parsing text-format caffe.NetParameter: 35:20: Message type "caffe.LayerParameter" has no field named "batch_norm_param".
F0309 13:19:36.307616 13339 upgrade_proto.cpp:928] Check failed: ReadProtoFromTextFile(param_file, param) Failed to parse NetParameter file: models/RESNET_50//train.prototxt

Do you know this problem?

**Comment by XiaozhiChen** [comment]
@peternara Seems your caffe did not support the batchNorm layer, or not the newest version. If you are using [py-faster-rcnn](https://github.com/rbgirshick/py-faster-rcnn), it should work.

**Comment by peternara** [comment]
thank you~!!^^
